.drone.yml

```diff
@@ -6237,7 +6237,94 @@ volumes:
     claim:
       name: drone-s3-debrepo-pvc
 ---
+kind: pipeline
+type: kubernetes
+name: publish-rlz
+
+environment:
+  RELCLI_BASE_URL: https://releases-staging.platform.teleport.sh
```

Comment:
Should not this be set in Drone environment?

Reply:
Not entirely sure what you meant by "Drone environment". It is true that this could be moved to the step `Publish in Release API`, as it is not required by multiple steps in the pipeline (unlike `RELCLI_IMAGE`).